rebar.config

```diff
@@ -1,37 +1,69 @@
+%% -*- mode: erlang; erlang-indent-level: 4; indent-tabs-mode: nil -*-
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2010-2016 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
 {src_dirs, ["./priv/tracers", "./src"]}.
+
 {cover_enabled, false}.
+
 {edoc_opts, [{preprocess, true}]}.
-{erl_opts, [warnings_as_errors,
-            {parse_transform, lager_transform},
-            {src_dirs, ["src", "priv/tracers"]},
-            {platform_define, "^[0-9]+", namespaced_types},
-            {platform_define, "^[0-9]+", set_env_options},
-            {platform_define, "^R15", "old_hash"},
-            {d, 'TEST_FS2_BACKEND_IN_RIAK_KV'}]}.
+
+{erl_opts, [
+    debug_info,
+    warnings_as_errors,
+    {parse_transform, lager_transform},
+    {src_dirs, ["src", "priv/tracers"]},
+    {platform_define, "^[0-9]+", namespaced_types},
+    {platform_define, "^[0-9]+", set_env_options},
+    {platform_define, "^R15", "old_hash"},
+    {d, 'TEST_FS2_BACKEND_IN_RIAK_KV'}
+]}.
 
 {eunit_opts, [
-     no_tty,  %% This turns off the default output, MUST HAVE
-     {report, {eunit_progress, [colored, profile]}} %% Use `profile' to see test timing information
-     %% Uses the progress formatter with ANSI-colored output
-     ]}.
+    %% This turns off the default output, MUST HAVE
+    no_tty,
+    %% Use the progress formatter with ANSI-colored output
+    %% Use `profile' to see test timing information
+    {report, {eunit_progress, [colored, profile]}}
+]}.
 
 {xref_checks, []}.
-%% XXX yz_kv is here becase Ryan has not yet made a generic hook interface for object modification
-%% XXX yz_stat is here for similar reasons -- we do not yet support dynamic stat hooks
-{xref_queries, [{"(XC - UC) || (XU - X - B - \"(cluster_info|dtrace|yz_kv|yz_stat)\" : Mod)", []}]}.
+{xref_queries, [
+    %% yz_kv because we don't have a generic hook interface for object modification
+    %% yz_stat for similar reasons -- we do not yet support dynamic stat hooks
+    {"(XC - UC) || (XU - X - B - \"(cluster_info|dtrace|yz_kv|yz_stat)\" : Mod)", []}
+]}.
 
 {erl_first_files, [
-                   "src/riak_kv_backend.erl"
-                  ]}.
+    "src/riak_kv_backend.erl"
+]}.
 
 {deps, [
```

Comment:
First, a general comment - I'd agree that it would be nice to have this set of changes in a separate PR. Making unrelated rebar.config changes in a PR like this makes it harder for this PR to move forward _and_ to get the rebar changes done - separate reviews would make this process smoother.

Second, to the actual changes:
Any changes that would allow a library that has on-disk format impacts to float on a branch is bad - thus, I agree that we shouldn't be floating sext on a branch, maybe ever. I understand that _this particular_ change won't, today, break anything if it's really only an edown change, but if the master branch changes (for example, if TS decides to make and merge a change we're not aware of), we could break Riak for users with existing data on-disk. To keep this PR moving, I'd be fine with any of the changes to the riak_* modules (that are pointing at develop-2.2 branches, as obviously the develop-2.2 branch of riak_kv wants those), sidejob (because I'm pretty sure we need to bump it anyway), and eper/eunit_formatters as debugging/unit testing tools, but I really don't want to see sext, or any other library that may cause on-disk incompatibility issues, to be floated on a branch at this point. That also means Bitcask should probably remain pinned to a tag so we can make conscious decisions about moving it, rather than getting "whatever's latest" and breaking something.